.github/stale.yml

```diff
@@ -0,0 +1,83 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+pulls:
+  # XXX: We'll only match very few issues to ensure the bot works as expected
```

Comment:
Why the `XXX`?

Reply:
Can use TODO instead. This is jsut to make clear that this configuration as such is for trying it out and not our real value